src/components/game/components/board/withBoardProps.js

```diff
@@ -49,6 +51,9 @@ const withBoardProps = (Component) => (props) => {
     gameStats: {
       pot: pot,
     },
+    logStats: {
+      logger: logger,
```

Comment:
```suggestion
      logger,
```
As from ES6 this can be writter like this way as well

Reply:
Done.